spec/outputs/pagerduty_spec.rb

```diff
@@ -1 +1,31 @@
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
+
+describe LogStash::Outputs::PagerDuty do
+
+  it "should register without errors" do
+    plugin = LogStash::Plugin.lookup("output", "pagerduty").new("service_key" => "")
+    expect { plugin.register }.to_not raise_error
+  end
+
+  describe "#send" do
+    subject          { LogStash::Outputs::PagerDuty.new( "service_key" => "my_service_key" ) }
+    let(:properties) {{ "message" => "This is a message!" }}
```

Comment:
related to spaces.. Why you are adding space after describe block and after end statement see https://github.com/logstash-plugins/logstash-output-pagerduty/pull/9/files#diff-c5f4c2492cfd323748a3212b3e74f54fR28 

I don't think they make code easier to read?